Data-Structures/Graph/Graph.js

```diff
@@ -1,14 +1,16 @@
 class Graph {
   constructor () {
     this.adjacencyMap = {}
+    this.numberOfVertex = 0
   }
 
   addVertex (vertex) {
     this.adjacencyMap[vertex] = []
+    this.numberOfVertex++
   }
 
   containsVertex (vertex) {
-    return typeof (this.adjacencyMap[vertex]) !== 'undefined'
+    return typeof this.adjacencyMap[vertex] !== 'undefined'
```

Comment:
Simply use `this.adjacencyMap[vertex] !== undefined`

Reply:
this code is not mine, but I'll make changes as you proposed...

Reply:
You can also undo all the changes to code that isn't yours.